### PR TITLE
Restaurant page, project on JS part of the Curriculum

### PR DESCRIPTION
**Pull request by Abdoulaye-Thespy**
Here we used Webpack bundler to create a Restaurant page with multiple JavaScript files.
Table navigation is used to switch between contents.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (lekegitrepo) [inline package.json]: - **_Commands like `build`, `development` should be in `package.json` inside \`scripts'. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))_**

```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

File: README.md [hunk 2]
```diff
@@ -1 +1,35 @@
-# Restaurant-Page
\ No newline at end of file
+# Restaurant-Page
+
+
+## [LiveDemo](https://abdoulaye-thespy.github.io/Restaurant-Page/#)
+
+## Project descipription
```
Comment (lekegitrepo) [inline README.md]: - **_Correct the typo error_**

File: README.md [hunk 3]
```diff
@@ -1 +1,35 @@
...
+
+- As this project is still under reviews the design is not quite it yet.
+
+- We have have an an app where the famous game Tic tac toe can be played using Vanilla javascript.
```
Comment (lekegitrepo) [inline README.md]: - **_Update the Readme with relevant information about the current project._**

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+const homePage = () => {
+  content.innerHTML = '';
+  pageLoad();
+};
+
+const Pagemenu = () => {
+  content.innerHTML = '';
+  menuPage();
+  menuBot.style.color = '#fff';
```
Comment (lekegitrepo) [inline src/index.js]: - **_Keep all the styling logic in CSS files_**

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Request changes :recycle:
### Hi Team :clap: :clap:
#### Good job so far :muscle: you have done a good job, however, you need to fix the linter errors and implement some missing requirement and indicated in my comments below.

- **_Fix the linter errors_**

**_Check below for the details of the requirements :point_down:_**

#### _Reviewed by Temitayo Adeleke._

**Review by trekab — CHANGES_REQUESTED** [review]
### Changes Required! :recycle: 
Hi @Abdoulaye-Thespy, well done implementing the requested changes. You have a few changes to make before moving forward though. These are:
- [ ] Necessary run common commands like `build`, `development` should be in `package.json` inside \`scripts'. This part is empty in your `package.json` file ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))
- [ ] Much as you have done a nice job customizing your `README.md` file, it could use some more improvements. Consider adding your app's screenshot to the `README.md` file. This would make your repository look more professional and encourage visitors to check out your app. Feel free to use [this template](https://github.com/microverseinc/readme-template)

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Review by Oghenebrume50 — APPROVED** [review]
### APPROVED

Hi @Abdoulaye-Thespy you have done really well, kudos on making changes this looks good to me, please merge and move on **This is approved**

**Raphael Noriode** :mask: STAY_SAFE

**Pull request merged**